rocAL/rocAL_pybind/examples/rocAL_api_video_pipeline.py

```diff
@@ -114,26 +120,24 @@ def main():
     display = args.display
     num_threads = args.num_threads
     random_seed = args.seed
-    tensor_format = types.NHWC if args.NHWC else types.NCHW
+    tensor_format = types.NFHWC if args.NHWC else types.NFCHW
     tensor_dtype = types.FLOAT16 if args.fp16 else types.FLOAT
     # Create Pipeline instance
     pipe = Pipeline(batch_size=batch_size, num_threads=num_threads,device_id=args.local_rank, seed=random_seed, rocal_cpu=_rocal_cpu,
-                    mean=[0.485 * 255, 0.456 * 255, 0.406 * 255], std=[0.229 * 255, 0.224 * 255, 0.225 * 255], tensor_layout=tensor_format, tensor_dtype=tensor_dtype)
+                    tensor_layout=tensor_format, tensor_dtype=tensor_dtype, output_memory_type=types.CPU_MEMORY if _rocal_cpu else types.GPU_MEMORY)
     # Use pipeline instance to make calls to reader, decoder & augmentation's
     with pipe:
         images = fn.readers.video(device="gpu", file_root=video_path, sequence_length=user_sequence_length,
                               normalized=False, random_shuffle=False, image_type=types.RGB,
                               dtype=types.FLOAT, initial_fill=16, pad_last_batch=True, name="Reader")
         crop_size = (512,960)
         output_images = fn.crop_mirror_normalize(images,
+                                            rocal_tensor_layout = tensor_format,
+                                            rocal_tensor_output_type = tensor_dtype,
                                             crop=crop_size,
                                             mean=[0, 0, 0],
-                                            std=[1, 1, 1],
-                                            mirror=0,
-                                            output_dtype=types.UINT8,
-                                            output_layout=types.NHWC,
-                                            pad_output=False)
-        pipe.set_outputs(output_images)
```

Comment:
I think this should still reamin set_outputs as this is how it is in DALI . The API thats exposed to the user should be kept set_output.

If needed, we can change all the APIs in pipeline.py to follow this convention.
@shobana-mcw please comment on this